### DISCUSSION TITLE
Being able to copy output image to clipboard

### DISCUSSION THREAD

**Discussion by joelpaula** [microsoft/vscode-jupyter#10093] [New Ideas]
A lot of times I am copying and pasting from Notebooks to emails, chats and slide decks. Why not make it simple and enable simple right click copy of image, instead of having to save to disk to be able to share the image?

**Comment by rchiodo** [comment]
This is a limitation of VS code at the moment (it's a missing API from the version of Electron that was in use). 

See this here:
https://github.com/microsoft/vscode/issues/129880
